lib/command/commandLine.js

```diff
@@ -887,6 +888,8 @@ var commands = {
 function init(cbCfg, iotCfg) {
     config = cbCfg;
     configIot = iotCfg;
+    if(!config.authorization && !config.authorization.header)
```

Comment:
The style we typically use is:

```
if (...) {
  ...
}
```

I suggest using the same here.

Reply:
fixed by 68f37d7